init.d/modules.BSD.in

```diff
@@ -0,0 +1,24 @@
+#!@SBINDIR@/openrc-run
+
+description="Loads a user defined list of kernel modules."
+
+depend() {
+	after bootmisc
+	keyword -jail -prefix
+}
+
+start()
+{
+	local cnt=0 x
+	for x in $modules
+	do
+		ebegin "Loading module $x"
+		kldload "$x";
+		eend $? "Failed to load $x" && : $(( cnt += 1 ))
```

Comment:
Just completely remove counting, and get rid of the "Autoloaded $x module(s)" message entirely.  I made the comment below before I got to this one.